language/apply.md

```diff
@@ -0,0 +1,34 @@
+Apply Expression
+===
+
+Apply is an expression intended to capture a manifest block, compile it to a catalog, and apply it on remote nodes.
```

Comment:
"An `ApplyExpression` describes a manifest block which will be compiled to a catalog and applied on remote nodes"